war/src/test/java/io/lumeer/engine/controller/search/QuerySuggesterIntegrationTest.java

```diff
@@ -188,6 +183,22 @@ public void testSuggestAttributesCompleteAttribute() throws Exception {
    }
 
    @Test
+   public void testSuggestAttributesChilds() throws Exception {
+      collectionFacade.createCollection(COLLECTION_COMPANIES);
+      String internalCollectionName = collectionMetadataFacade.getInternalCollectionName(COLLECTION_COMPANIES);
+      collectionMetadataFacade.addOrIncrementAttribute(internalCollectionName, "Lumeer");
+      collectionMetadataFacade.addOrIncrementAttribute(internalCollectionName, "First.Lumeer");
+      collectionMetadataFacade.addOrIncrementAttribute(internalCollectionName, "Second.First.Lumeer");
+      collectionMetadataFacade.addOrIncrementAttribute(internalCollectionName, "a.b.c.d.e.Lumee");
+
+      List<SearchSuggestion> attributes = querySuggester.suggestAttributes("companies.lumee", QuerySuggester.SUGGESTIONS_LIMIT);
+      assertThat(attributes).hasSize(4);
```

Comment:
Hmmm I think that it's not necessary, because structure is checking in other tests, but I can do that to make sure everything works fine